packages/gollm/gollm_openai/tool_utils.py

```diff
@@ -270,7 +270,7 @@ def amr_enrichment_chain(amr: str, research_paper: str) -> dict:
 
     print("Building prompt to extract model enrichments from a research paper...")
     prompt = ENRICH_PROMPT.format(
-        amr=escape_curly_braces(amr),
+        amrs=[escape_curly_braces(amr) for amr in amrs],
```

Comment:
I am not seeing any big changes to the prompt. 
Is this PR to setup up the plumbing only?
Are there any example results we can take a look at?